syntax = "proto3";

import "student.proto";

package students;

message StudentsGroup
{
	repeated Student students = 1;
}